src/lambdas/api/app.js

```diff
@@ -179,21 +191,22 @@ app.put('/collections/:collectionId/items/:itemId', async (req, res, next) => {
     } else if (req.body.id && req.body.id !== itemId) {
       next(createError(400, 'Item ID in resource URI must match id in body'))
     } else {
-      const collectionRes = await api.getCollection(collectionId, es, req.endpoint)
-      if (collectionRes instanceof Error) next(createError(404))
-
-      req.body.collection = collectionId
-      req.body.id = itemId
-      try {
-        await api.updateItem(req.body, es)
-        res.sendStatus(204)
-      } catch (error) {
-        if (error instanceof Error
-              && error.name === 'ResponseError'
-              && error.message.includes('version_conflict_engine_exception')) {
-          res.sendStatus(409)
-        } else {
-          next(error)
+      const itemRes = await api.getItem(collectionId, itemId, es, req.endpoint)
```

Comment:
validate the item exists for a put rather than just the collection (should have been pulled into another PR, but here we are)